Integrations/Cloaken/Cloaken.py

```diff
@@ -58,9 +61,9 @@ def get_client():
     elif response_code == 400:
         # url was malformed
         context = {
-            'original_url': url,
-            'unshortened_url': '',
-            'status': response_code
+            'OriginalURL': url,
```

Comment:
this might cause a backward compatible issues.
please keep the original keys in addition to the fixed keys (same for the other two keys):
```suggestion
            'OriginalURL': url,
            'original_url': url,
```